_layouts/docwithnav.html

```diff
@@ -16,7 +16,12 @@
 	<div class="row">
 		<!-- Navigation menu -->
 		<div class="col-md-3 hidden-sm hidden-xs" id="wrapper">
-			{% include nav_v1.0.html %}
+			{% case page.collection %}
```

Comment:
this adds new _v1.1 content into the build and specifies what version of the nav menus to use per "collection" (ie _v1.0 or _v1.1)